Utilities/CodeGeneration/EdFi.Ods.CodeGen/Options.cs

```diff
@@ -21,5 +22,8 @@ public class Options
 
         [Option('i', "includePlugins", Required = false, HelpText = "Flag to determine if plugin assemblies are included", Default = false)]
         public bool IncludePlugins { get; set; }
+
+        [Option('j', "extensionsPaths", Required = false, HelpText = "Array of paths for the Extension location to determine if plugin assemblies are included")]
+        public IEnumerable<string> ExtensionsPaths { get; set; }
```

Comment:
Should this be an immutable object?

Reply:
it should be IEnumerable<string> for any array with CommandLine